README.md

```diff
@@ -4,11 +4,16 @@ The Student Robotics public style guide.
 
 ## Getting Started
 
-1. [Install Ruby][install-ruby]
+1. [Install Ruby 3][install-ruby]
```

Comment:
nit: This is kind of redundant.

Reply:
Eh, true. Do you think we should remove it? (This was copied from the website repo, so it'd be good to update all the READMEs if we think we don't want this detail)

Reply:
Doesn't really matter.